### DISCUSSION TITLE
Share all folders between 2 users

### DISCUSSION THREAD

**Discussion by umeshauv** [drakkan/sftpgo#1392] [General]
Hi 
  I have a scenario where I need to share folders between 2 users.  They need to access the same directories, one with both read/write permission and another with read-only.   Also, if the first user creates new directories, they should be sharable between them.  This seems to be a simple case of sharing, but not finding information as to how to do.  Can you help ?

Thanks & Regards,
Umesh

**Comment by JonBendtsen** [comment]
I think you should create a virtual folder, and grant different access rights to that virtual folder.

**Comment by umeshauv** [comment]
I was able to do it by creating a new user with home directory set to home directory of first user.

**Reply by Joshua2504** [reply]
Hi,

can you confirm that this is working flawless in regards of user permissions etc?

Regards

**Reply by umeshauv** [reply]
Yes.  It's working fine for me and permissions are different for each user
